.github/workflows/duplicate_samples.yml

```diff
@@ -0,0 +1,49 @@
+name: Duplicate Package Samples Workflow
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  duplicate:
+    name: Duplicate samples job 👯
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - name: Duplicate Web3 Unity Samples
+      run: |
+        for i in 1 2
+        do
+          export SOURCE_PATH="importable_sample_path_$i"
+          SOURCE_PATH=${!SOURCE_PATH}
+          
+          export DESTINATION_DIRECTORY="importable_sample_directory_$i"
+          DESTINATION_DIRECTORY=${!DESTINATION_DIRECTORY}
+          
+          SOURCE_PATH="${GITHUB_WORKSPACE}/$SOURCE_PATH/$(ls "$SOURCE_PATH" | head -n 1)/$DESTINATION_DIRECTORY"
+        
+          export DESTINATION_PATH="imported_sample_path_$i"
+          DESTINATION_PATH=${!DESTINATION_PATH}
+          DESTINATION_PATH="${GITHUB_WORKSPACE}/$DESTINATION_PATH"
+          
+          rm -rf "$DESTINATION_PATH"
+          mkdir -p "$DESTINATION_PATH"
+          cp -r "$SOURCE_PATH" "$DESTINATION_PATH"
+        done
+      env: 
```

Comment:
it would be amazing if we can somehow copy and paste all the packages samples like that. So that even if we add dozens of packages we do not have to modify the script at all. I am not a CI expert so don't know if it's doable.

Reply:
@robGG1997 I think the whole thing can be simplified and also meet the expectations Nikola brought.

1. We can define environment variable in the repository settings that looks like an array of sources and destinations separated by cologne. Smth like `SAMPLES_DIRS=(src1:dst1 src2:dst2 src3:dst3)`
2. Iterate over this array via `for entry in "${!SAMPLES_DIRS[@]}"`
3. Split each entry into pair `IFS=':' read -ra dirs <<< "$entry"`
4. And then do a copy `cp -r dirs[0] dirs[1]`

So from this moment on, the only thing you need to update is the value of `SAMPLES_DIRS`

Reply:
This vars I mean
<img width="1227" alt="image" src="https://github.com/ChainSafe/web3.unity/assets/1240189/6e7869a0-ff4d-4168-bfcf-0157808fe53a">

Reply:
Understood, I'll update the CI based on this

Reply:
Updated! this is a much cleaner solution, thanks @kalambet